Added feature flag for send message (#761)
obsidiandynamics/kafdrop: README.md

--topic.createEnabled=false
```

## Message Configuration

> By default, you couldn't send messages to a topic. If you want to enable this feature, you could enable it with:

Reviewer: ```suggestion
By default, you cannot send messages to a topic. You can enable it with:
```

Author: I'm addressed it
